src/library-configuration/ethereum-providers.md

```diff
@@ -0,0 +1,94 @@
+# Ethereum Providers
+
+Resolution libraries require a connection to the Ethereum network to resolve .crypto and .eth domains. To initialize the library, you need to specify an Ethereum node service provider. Once the instance is created you can begin resolving domains. Below are examples of how to initialize the library with different providers.
+
+### Import library
```

Comment:
Let's split the article into 3 big categories: 
- javascript / typescript
- java
- swift

Each category should have `import + configuration` examples tight together. For example:

## Javascript / Typescript

```
import Resolution from "@unstoppabledomains/resolution";

const infuraApiKey = INFURA_PROJECT_ID;

 const resolution = new Resolution({
   cns: {
     url: `https://mainnet.infura.io/v3/${infuraApiKey}`,
     network: "mainnet",
   },
 });

 // or

 const resolution = Resolution.infura(infuraApiKey, "mainnet");
```